### PR TITLE
Upgrade jQuery in sample to version 3.5.1

### PR DESCRIPTION
**Pull request by wenz**
Update all jQuery files and references to them to version 3.5.1

Addresses #21572

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
Thanks for your PR, @wenz.
@captainsafia can you please review this and merge if things look good? Thanks!

File: src/Azure/AzureAD/samples/AzureADB2CSample/Pages/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -51,11 +51,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Azure/AzureAD/samples/AzureADB2CSample/Pages/Shared/_Layout.cshtml:58]: SRI hash looks good here.

File: src/Azure/AzureAD/samples/AzureADB2CSample/wwwroot/lib/jquery/.bower.json [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
-  "version": "3.4.1",
-  "_release": "3.4.1",
+  "version": "3.5.1",
+  "_release": "3.5.1",
   "_resolution": {
     "type": "version",
-    "tag": "3.4.1",
-    "commit": "15bc73803f76bc53b654b9fdbbbc096f56d7c03d"
+    "tag": "3.5.1",
+    "commit": "4c0e4becb8263bb5b3e6dadc448d8e7305ef8215"
```
Comment (captainsafia) [inline src/Azure/AzureAD/samples/AzureADB2CSample/wwwroot/lib/jquery/.bower.json:20]: Commit hash looks good here.

File: src/Azure/AzureAD/samples/AzureADSample/Pages/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -51,11 +51,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Azure/AzureAD/samples/AzureADSample/Pages/Shared/_Layout.cshtml:58]: SRI hash looks good here.

File: src/Security/samples/StaticFilesAuth/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -51,11 +51,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Security/samples/StaticFilesAuth/Views/Shared/_Layout.cshtml:58]: Hash looks good here.

File: src/Security/samples/PathSchemeSelection/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -54,11 +54,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Security/samples/PathSchemeSelection/Views/Shared/_Layout.cshtml:61]: Hash looks good.

File: src/Security/samples/Identity.ExternalClaims/Pages/_Layout.cshtml [hunk 6]
```diff
@@ -52,11 +52,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Security/samples/Identity.ExternalClaims/Pages/_Layout.cshtml:59]: Hash looks good.

File: src/Security/samples/DynamicSchemes/Pages/_Layout.cshtml [hunk 7]
```diff
@@ -51,11 +51,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Security/samples/DynamicSchemes/Pages/_Layout.cshtml:58]: Hash looks good.

File: src/Security/samples/Cookies/Views/Shared/_Layout.cshtml [hunk 8]
```diff
@@ -54,11 +54,11 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery"
                 crossorigin="anonymous"
-                integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=">
+                integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=">
```
Comment (captainsafia) [inline src/Security/samples/Cookies/Views/Shared/_Layout.cshtml:61]: Hash looks good.

**Review by captainsafia — APPROVED** [review]
Hashes and file contents look good on this. Thank you.

**Pull request merged**